src/com/company/Main.java

```diff
@@ -12,14 +12,14 @@ public static void main(String[] args){
 
         String txtfile = args[0];
         File file = new File(txtfile);
-        checkforsupport(file);
-        checkforexistance(file);
+        checkForSupport(file);
+        checkForExistance(file);
```

Comment:
Might be good if we change the method name lie checkIfFileExists(arguments)

Reply:
I have changed all the method names.